assets/_scss/components/_disclaimer.scss

```diff
@@ -1,23 +1,23 @@
 .usa-disclaimer {
-  @include media($small-screen) {
-    font-size: $h5-font-size;
-  }
   background-color: $color-gray-lightest;
   font-size: $h6-font-size;
-  padding: {
-    bottom: .5rem;
-    top: .5rem;
+  padding-bottom: .5rem;
+  padding-top: .5rem;
+
+  @include media($small-screen) {
+    font-size: $h5-font-size;
   }
 }
 
 // This hides the stage note (i.e. alpha, beta, etc) of the disclaimer for 
 // anything smaller than large screen sizes
 .usa-disclaimer-stage {
+  display: none;
```

Comment:
A screen reader on a desktop would read it and on a mobile device would not. I don't know what the priority of the stage text is for screen reader users.